### ISSUE TITLE
[Mobile] You cannot type with a physical keyboard on Android and iOS devices.

### ISSUE THREAD

**Issue by tony-soft** [singerdmx/flutter-quill#1386]
My issue is about [Mobile]

To test this issue you will need a physical keyboard, such as a Bluetooth keyboard.

Bug details:
1. When typing with a physical keyboard, only the first letter can be typed and no additional typing is possible after that.
2. If you are unable to type, press Backspace to enable typing again.

This is a video I shot.

https://github.com/singerdmx/flutter-quill/assets/70277412/fcf2683e-1dfb-40ae-801e-5e1dff3f5e95

I typed additionally after the first typing using the Bluetooth keyboard, but it is not reflected on the UI.
And if I press backspace on the Bluetooth keyboard to delete a single letter, I can only type one letter again.
I tested it using the basic example you provided.

I am using flutter_quill version 7.4.4 and flutter is using version 3.13.1.

**Comment by tony-soft** [comment]
We would like to inform you of additional details we have confirmed.
I'm using a `Logitech K780` Bluetooth keyboard.

There was a problem with the iPhone in the previous version, but it appears to have been completely resolved now. Sorry for the confusion.😢

The problem occurs on Android phones, but I think it may be a problem with the keyboard app.
The test device I use is `Samsung Galaxy S20 5G (SM-G981N)`.
I think that on Samsung phones and tablets, the keyboard is set to use the Samsung keyboard app by default.
<img width="572" alt="스크린샷 2023-09-01 오전 6 00 08" src="https://github.com/singerdmx/flutter-quill/assets/70277412/42ee5402-8246-4372-a960-82253e9e5af1">


After hearing that you are not experiencing any problems, I installed the `Google Keyboard (Gboard)` app and tested it.
The result was successful. The problem I experienced does not occur with Gboard.

I think this is very good news.
However, an issue that should not be overlooked is that Samsung mobile phones have a high market share.
